ui/ducks/send/send.js

```diff
@@ -2834,6 +2850,7 @@ export function signTransaction() {
           sendAsset: { ...sendAsset },
           receiveAsset: { ...receiveAsset },
           recipient: { ...recipient },
+          amountMode: state[name].amountMode,
```

Comment:
Not related to this commit - can we disable the Confirm button when Swap + send-ing with 0 amount? It's supported for simple Send (useful for cancelling txs, etc), but I don't see a usecase for using a 0-amount swap trade